webapp/components/button.tsx

```diff
@@ -1,4 +1,9 @@
+import Link from 'next-intl/link'
 import { ComponentProps } from 'react'
+import { isRelativeUrl } from 'utils/url'
+
+const commonCss = `text-ms box-content flex items-center justify-center
```

Comment:
Do we need backticks here? There is no string interpolation.

Reply:
it is needed to use multiline strings!

<img width="1077" alt="image" src="https://github.com/user-attachments/assets/d6b5892b-afe8-4dad-9b55-2ec4d6395583">


as it is a long string, I felt it read better in two lines. (It's a 37 length string(